### ISSUE TITLE
Bottom border cut off

### ISSUE THREAD

**Issue by ssashok10** [kekeh/mydatepicker#65]
I am using mydatepicker in my Angular2 application and i am not seeing the bottom border of this control. I am not using any custom CSS to modify the style of it. Is there any CSS class that i need to use to see the bottom border of this control? I am using it as mentioned below,

```
<my-date-picker [options]="myDatePickerNormalOptions" (dateChanged)="onDateChanged($event)"></my-date-picker>
```

**Comment by kekeh** [comment]
I fixed bottom border visibility issue. Try the latest version: 1.0.2.

**Comment by ssashok10** [comment]
Thanks. The issue is fixed with the version 1.0.2
